src/paperbot/api/routes/studio_chat.py

```
@@ -0,0 +1,417 @@
+                "content": full_content,
+                "mode": request.mode,
+                "model": request.model,
+            }
+        )
+
+    except FileNotFoundError:
+        yield StreamEvent(
+            type="error",
+            message=f"Claude CLI not found at: {claude_path}"
+        )
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)